_scalingo_api/domains.md

```diff
@@ -95,7 +100,10 @@ Returns 200 OK
             "tlscert": "/C=FR/ST=Some-State/O=Internet Widgits Pty Ltd/CN=example.com",
             "tlskey": "RSA private key - 2048 bytes",
             "validity": "2015-08-05T19:57:21.000+02:00",
-            "canonical": false
+            "canonical": false,
+            "letsencrypt_enabled": true,
+            "letsencrypt": false,
+            "letsencrypt_status": "created"
```

Comment:
Shouldn't be 
```suggestion
            "letsencrypt": true,
            "letsencrypt_status": "created"
```
if `letsencrypt_status` is created?

Because we should have the cert already and be able to use LetsEncrypt for this domain

Reply:
Actually no, we could have the `created` status (meaning we have generated a let's encrypt)
BUT a customer certificate is set, so `letsencrypt` is false, which means: is not used